### PR TITLE
[13.0][MIG] base_optional_quick_create

### PR DESCRIPTION
**Pull request by baimont**

### CODE REVIEW

**Comment by Reyes4711-S73** [comment]
Functional review

**Review by len-foss — APPROVED** [review]
Code review

File: base_optional_quick_create/models/ir_model.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    @api.model_create_multi
+    def create(self, vals_list):
+        ir_models = super().create(vals_list)
+        ir_models._patch_quick_create()
+        return ir_models
+
+    def write(self, vals):
+        res = super().write(vals)
+        self._patch_quick_create()
+        return res
```
Comment (okuryan) [inline base_optional_quick_create/models/ir_model.py:55]: Should this be included also? https://github.com/OCA/server-ux/pull/318

Comment (vanderperre) [reply base_optional_quick_create/models/ir_model.py:55]: ok

**Review by acsonefho — COMMENTED** [review]
Code LGTM and seems working on my project :)

**Comment by ThomasBinsfeld** [comment]
@dreispt @thomaspaulb @pedrobaeza @sebalix is it possible to reopen this PR and merge it? It is approved and used in production environment for a year now.

**Comment by ThomasBinsfeld** [comment]
Thanks @pedrobaeza !

**Pull request merged**